src/XliffTasks.Tests/XamlRuleDocumentTests.cs

```diff
@@ -47,28 +119,50 @@ public void BasicLoadAndTranslate()
 
             string expectedTranslation =
 @"<Rule Name=""MyRule"" DisplayName=""AAA"" PageTemplate=""generic"" Description=""BBB"" xmlns=""http://schemas.microsoft.com/build/2009/properties"">
+  <!-- DisplayName: My rule display name comment -->
+  <!-- Description: My rule description comment -->
   <Rule.Categories>
-    <Category Name=""MyCategory"" DisplayName=""CCC"" />
+    <Category Name=""MyCategory"" DisplayName=""CCC"">
+      <!-- DisplayName: My category display name comment -->
+    </Category>
   </Rule.Categories>
   <EnumProperty Name=""MyEnumProperty"" DisplayName=""DDD"" Category=""MyCategory"" Description=""EEE"">
-    <EnumValue Name=""First"" DisplayName=""FFF"" />
+    <!-- DisplayName: My enum property display name comment -->
+    <!-- Description: My enum property description comment -->
+    <EnumValue Name=""First"" DisplayName=""FFF"">
+      <!-- DisplayName: My first item comment -->
+    </EnumValue>
     <EnumValue Name=""Second"" DisplayName=""GGG"" />
     <EnumValue Name=""Third"" DisplayName=""HHH"" />
   </EnumProperty>
   <BoolProperty Name=""MyBoolProperty"" Description=""III"" />
   <StringProperty Name=""MyStringProperty"">
     <StringProperty.Metadata>
-      <NameValuePair Name=""SearchTerms"" Value=""JJJ"" />
+      <NameValuePair Name=""SearchTerms"" Value=""JJJ"">
+        <!-- Value: My search terms comment -->
+      </NameValuePair>
     </StringProperty.Metadata>
   </StringProperty>
 </Rule>";
 
             var document = new XamlRuleDocument();
             var writer = new StringWriter();
             document.Load(new StringReader(source));
+
+            var xliffDocument = new XlfDocument();
+            xliffDocument.LoadNew("fr");
```

Comment:
what is the string param here (and why that one)?

Reply:
It's a language code, for French. Copied from `XlfDocumentTests`.